lib/ueberauth/strategy/github.ex

```diff
@@ -163,7 +159,7 @@ defmodule Ueberauth.Strategy.Github do
     %Info{
       name: user["name"],
       nickname: user["login"],
-      email: user["email"] || Enum.find(user["emails"] || [], &(&1["primary"]))["email"],
+      email: user["email"] || get_primary_email!(user),
```

Comment:
Should we create a `fetch_email` function for this code?  I noticed it's re-used in the `fetch_uid/2` function.

```elixir
defp fetch_email(user), do: user["email"] || get_primary_email!(user)
```